docs/APIReviewProcess.md

```diff
@@ -0,0 +1,71 @@
+## Description
+Starting in 5.0, certain areas within the dotnet/aspnetcore and dotnet/extensions repos will require formal *incremental* API reviews before any PRs that change APIs are merged.
+
+API changes to the following areas are required to go follow this process:
+
+* area-azure
+* area-hosting
+* area-installers
+* area-middleware
+* area-mvc
+    * feature-model-binding
+    * feature-razor-pages
+    * feature-JSONPatch
+    * feature-discovery
+    * feature-formatters
+    * feature-api-explorer
+    * feature-tag-helpers
+* area-security
+* area-servers
+* area-signalr
+* area-websockets
```

Comment:
If there's no opposition, I'd also be OK with flipping this to be a "Areas that **don't** need incremental API review" list. What would that look like?

* Blazor?
* Other MVC areas?

At the moment, there isn't really an area in my team that I wouldn't put through incremental API review. Most are pretty mature.

Reply:
Flipping this loosens up the rule as somebody can interpret it as "These are not required and nobody claims that what I am working on is required too. Let's just have a `white-list` approach".

Reply:
Sure I'm OK with that.